doc/2/plugins/plugin-context/accessors/sdk/index.md

```diff
@@ -69,13 +69,72 @@ async myAwesomePipe (request) {
 
 - The created document will have the `author` metadata property set to `null`.
 
+## Realtime notifications
+
+<SinceBadge version="change-me" />
+
+The [realtime](/sdk/js/7/controllers/realtime) is entirely available with the [subscribe](/sdk/js/7/controllers/realtime/subscribe) and [unsubscribe](/sdk/js/7/controllers/realtime/unsubscribe) methods.  
+
+Realtime subscription should be made in the plugin [init](core/2/plugins/guides/manual-setup/init-function) method or in a hook on the [kuzzle:start:before](/core/2/plugins/guides/events/kuzzle-start) event.
+
+::: warning
+If you want to make subscription in hooks, pipes, or controller actions then you should use the `replicate: true` option otherwise your callback will not be executed with every notification.
```

Comment:
Hard to understand, and possibly a source of many support tickets.

I have multiple possible suggestions to improve the wording of this warning, but first I need to understand something that bothers me:
* when `replicate: true` is set, what is replicated? The subscription, or notifications resulting from it?
* in the latter case, will a request submitted on a different node:
  * trigger the notification on the node where the plugin made the subscription?
  * trigger the notification on the same node than where the notification is generated (the subscription is replicated but the notification is not propagated)?
  * don't trigger the notification

Reply:
Do you still need insight on this one since we discuss the matter ?

Reply:
Nope I'm fine now. Thanks.